src/core/importers/specs.ts

```diff
@@ -36,6 +36,13 @@ export const importSpecs = async (
     : parserDefaultOptions;
   const data = (await SwaggerParser.resolve(path, parserOptions)).values();
 
+  // normalizing slashes after SwaggerParser
+  Object.keys(data).forEach(key => {
+    const value = data[key];
+    delete data[key];
+    data[resolve(key)] = value;
+  });
+  
```

Comment:
```suggestion
const resolveSpecs = async (
  path: string,
  parserOptions: SwaggerParser.Options,
) => {
  const data = (await SwaggerParser.resolve(path, parserOptions)).values();

  // normalizing slashes after SwaggerParser
  return Object.fromEntries(
    Object.entries(data).map(([key, value]) => [resolve(key), value]),
  );
};
```
If you want to use the Object.fromEntries you need to update the tsconfig lib also change `es2015` to `es2019`

I would prefer something like this. What do you think?

Reply:
Nice solution! 
I do still have some issues with codestyle, when 3+ actions are inlined and you need to parse long lines by eyes, instead of quick - "1-2 actions per line" falling down. Anyway, that sticks to the whole project codestyle. Also using of ->entries-> is consistent. So yes, sure!